pkg/logcli/query/query.go

```diff
@@ -259,17 +259,6 @@ func (q *Query) isInstant() bool {
 }
 
 func (q *Query) printStream(streams loghttp.Streams, out output.LogOutput, lastEntry []*loghttp.Entry) (int, []*loghttp.Entry) {
-	common := commonLabels(streams)
```

Comment:
Looks like we already skip printing common labels in Quiet mode https://github.com/grafana/loki/pull/4522/files#diff-1a68a4d189f6dc4eda3cffe8ba6735d4480c95fedde0ba9dc167adba3c48cc44L269
Was it not working before?

Reply:
@sandeepsukhani yes I can suppress the common labels in Quiet-mode - this does still work. But that's not what I want here. I instead want to ensure that **all** labels are present in the output on STDOUT, so that I can pipe the output from logcli into other command-line programs. See the original PR description and the issue referenced. 

Basically I want logcli to adhere more closely to the Unix philosophy. I'll use other unix programs downstream in the pipe as I see fit to further edit and filter log lines - I don't want logcli to be "too clever" here.